examples/ultralytics-yolo/deepsparse_utils.py

```diff
@@ -549,6 +550,30 @@ def download_model_if_stub(path: str) -> str:
     return path
 
 
+def download_pytorch_model_if_stub(path: str) -> str:
+    """
+    Utility method to download PyTorch model if path is a SparseZoo stub
+
+    :param path: file path to YOLO .pt model or SparseZoo stub of the model
+        to be loaded
+    :return: filepath to the .pt model
+    """
+    if path.startswith("zoo"):
+        model = Zoo.load_model_from_stub(path)
+        downloded_path = model.onnx_file.downloaded_path()
+        downloaded_pt_path = None
+        for file in model.framework_files:
+            if file.file_type_framework and file.display_name == "model.pt":
+                downloaded_pt_path = file
```

Comment:
might have missed something, but I believe we still need to download the file here

Reply:
i thought accessing `framework_files` would auto-download but i will make sure here